TestProjects/VisualEffectGraph_URP/Assets/ForwardRenderer.asset

```diff
@@ -9,7 +9,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 0}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: f971995892640ec4f807ef396269e91e, type: 3}
+  m_Script: {fileID: 11500000, guid: de640fe3d0db1804a85f9fc8f5cadab6, type: 3}
```

Comment:
See this [thread](https://unity.slack.com/archives/GBJ7AU4V8/p1631798764036800) and this [backport PR](https://github.com/Unity-Technologies/Graphics/pull/5687)
Correct me if I am wrong but this change is also expected on `2021.2/staging` ?
This PR https://github.com/Unity-Technologies/Graphics/pull/5410 landed just before the 21.2 branch off.

Reply:
yup, will have to back port it quickly too to make sure less random failures for Yamato tests fro everyone